public/app/core/services/keybindingSrv.ts

```diff
@@ -196,7 +197,7 @@ export class KeybindingSrv {
     });
 
     // jump to explore if permissions allow
-    if (this.contextSrv.isEditor && config.exploreEnabled) {
+    if ((this.contextSrv.isEditor || config.viewersCanEdit) && config.exploreEnabled) {
```

Comment:
think a until function on contextSrv named canAccessExplore() that checks role & viewsersCanEdit & config.exploreEnabled flag would be nice. then it can be reused in the metric_panel_ctrl if statement as well.

Reply:
Will fix!